pagopa/io-app
# fix: [IA-918] Android 12 local push notification app crash fix

## Short description
This PR is going to definitely fix the crash happening in Android 12 during the very first application run. This crash happens because there are not enough permissions to run local push notifications. The fix implemented by this PR is just a check to avoid the crash.

This PR is related to https://github.com/pagopa/io-app/pull/4060.

## List of changes proposed in this pull request
- Check whether there are enough permissions to schedule a local push notification during the very first application run.

## How to test
Using an Android 12 device, Install the application, remove all the app's permissions (notifications, reminders, ...) and run it. The app should not crash.
